[docs] minor improvements for system-views.md (#22724)
ydb-platform/ydb: ydb/docs/en/core/devops/observability/system-views.md

| ExpectedSlotCount     | Uint32    |         | Maximum number of VSlots that can be created on this PDisk. Either user-defined or inferred from `InferPDiskSlotCountFromUnitSize`.                                |
| NumActiveSlots        | Uint32    |         | Number of currently occupied VSlots with respect to GroupSizeInUnits of VDisks                                                                                   |
| SlotSizeInUnits       | Uint32    |         | Size of VSlot in abstract units. Either user-defined or inferred from `InferPDiskSlotCountFromUnitSize`.                                                           |
| DecommitStatus        | String    |         | Status of PDisk [decommissioning](../deployment-options/manual/decommissioning.md) (`DECOMMIT_NONE`, `DECOMMIT_PENDING`, `DECOMMIT_IMMINENT`, `DECOMMIT_REJECTED`)       |
| InferPDiskSlotCountFromUnitSize  | Uint64    |         | Size of VSlot in bytes from which `ExpectedSlotCount` and `SlotSizeInUnits` values are inferred unless user-defined. |

> The inferred values of `ExpectedSlotCount` and `SlotSizeInUnits` are defined by the formula $\text{ExpectedSlotCount} \times \text{SlotSizeInUnits} = \frac{\text{TotalSize}}{\text{InferPDiskSlotCountFromUnitSize}}
> The inferred values of `ExpectedSlotCount` and `SlotSizeInUnits` are defined by the formula $\text{ExpectedSlotCount} \times \text{SlotSizeInUnits} = \frac{\text{TotalSize}}{\text{InferPDiskSlotCountFromUnitSize}}$, where $\text{SlotSizeInUnits} = 2^N$ is chosen to meet $\text{ExpectedSlotCount} \leq 16$.

Author: ```suggestion
The inferred values of `ExpectedSlotCount` and `SlotSizeInUnits` are defined by the following formula:

$$
\text{ExpectedSlotCount} \times \text{SlotSizeInUnits} = \frac{\text{TotalSize}}{\text{InferPDiskSlotCountFromUnitSize}}
$$

$\text{SlotSizeInUnits} = 2^N$ is chosen to meet $\text{ExpectedSlotCount} \leq 16$.
```

| AvailableSize         | Uint64    |         | Number of bytes available for allocation on PDisk                                                                                                                |
| TotalSize             | Uint64    |         | Total number of bytes on PDisk                                                                                                                                   |
| Status                | String    |         | PDisk operating mode that affects its participation in group allocation (`ACTIVE`, `INACTIVE`, `BROKEN`, `FAULTY`, `TO_BE_REMOVED`)                                       |
| StatusChangeTimestamp | Timestamp |         | Time when `Status` last changed; if `NULL`, `Status` has not changed since PDisk creation                                                                              |
| ExpectedSlotCount     | Uint32    |         | Maximum number of VSlots that can be created on this PDisk. Either user-defined or inferred from `InferPDiskSlotCountFromUnitSize`.                                |

> | NumActiveSlots        | Uint32    |         | Number of currently occupied VSlots with respect to GroupSizeInUnits of VDisks                                                                                   |

Author: ```suggestion
| NumActiveSlots        | Uint32    |         | Number of currently occupied VSlots with respect to `GroupSizeInUnits` of VDisks                                                                                   |
```

| SlotSizeInUnits       | Uint32    |         | Size of VSlot in abstract units. Either user-defined or inferred from `InferPDiskSlotCountFromUnitSize`.                                                           |
| DecommitStatus        | String    |         | Status of PDisk [decommissioning](../deployment-options/manual/decommissioning.md) (`DECOMMIT_NONE`, `DECOMMIT_PENDING`, `DECOMMIT_IMMINENT`, `DECOMMIT_REJECTED`)       |
| InferPDiskSlotCountFromUnitSize  | Uint64    |         | Size of VSlot in bytes from which `ExpectedSlotCount` and `SlotSizeInUnits` values are inferred unless user-defined. |

The inferred values of `ExpectedSlotCount` and `SlotSizeInUnits` are defined by the following formula:

$$
\text{ExpectedSlotCount} \times \text{SlotSizeInUnits} = \frac{\text{TotalSize}}{\text{InferPDiskSlotCountFromUnitSize}}
$$

> $\text{SlotSizeInUnits} = 2^N$ is chosen to meet $\text{ExpectedSlotCount} \leq 16$.

Author: ```suggestion
Where $\text{SlotSizeInUnits} = 2^N$ is chosen to meet $\text{ExpectedSlotCount} \leq 16$.
```

| SeenOperational     | Bool     |         | Boolean flag showing whether the group was in operational state after its creation                        |
| PutTabletLogLatency | Interval |         | 90th percentile of `PutTabletLog` request execution time                                                    |
| PutUserDataLatency  | Interval |         | 90th percentile of `PutUserData` request execution time                                                     |
| GetFastLatency      | Interval |         | 90th percentile of `GetFast` request execution time                                                         |
| OperatingStatus     | String   |         | Group status based on latest VDisk reports only (`UNKNOWN`, `FULL`, `PARTIAL`, `DEGRADED`, `DISINTEGRATED`)         |
| ExpectedStatus      | String   |         | Status based not only on operational report, but on PDisk status and plans too (`UNKNOWN`, `FULL`, `PARTIAL`, `DEGRADED`, `DISINTEGRATED`) |
| GroupSizeInUnits    | Uint32   |         | Size of the group in abstract units. In proportion to it, VDisks receive a storage quota. |

> The number of VSlots occupied by VDisk is defined as $\left\lceil \frac{\text{VDisk.GroupSizeInUnits}}{\text{PDisk.SlotSizeInUnits}} \right\rceil$.

Author: ```suggestion
The number of VSlots occupied by VDisk is defined as $ceil(\frac{\text{VDisk.GroupSizeInUnits}}{\text{PDisk.SlotSizeInUnits}})$.
```
